boards/common/stm32/dist/stm32f7.cfg

```diff
@@ -1,3 +1,6 @@
 source [find target/stm32f7x.cfg]
+
+flash bank $_FLASHNAME.noprobe stm32f2x 0x08000000 0 0 0 $_TARGETNAME
```

Comment:
Why `stm32f2x` ? Shouldn't it be `stm32f7x` instead ?

Reply:
To be honest I just used the same config that was in the openocd file, but replacing the address. I'll take a close look at all the parameters.

Reply:
Ok, had a look `stm32f7` doesn't have a specific flash driver they use `stm32f2x`, see files in https://github.com/ntfreak/openocd/tree/master/src/flash/nor and have a look at

https://github.com/ntfreak/openocd/blob/f98099507f509db9a18c70365490a6b9f67108db/src/flash/nor/stm32f2x.c#L67-L83